src/databricks/labs/remorph/snow/lca_utils.py

```diff
@@ -13,15 +14,15 @@
 
 
 def check_for_unsupported_lca(
-    dialect: str,
+    dialect: DialectType,
     sql: str,
     filename: str,
 ) -> ValidationError | None:
     """
     Check for presence of unsupported lateral column aliases in window expressions and where clauses
     :return: An error if found
     """
-    if dialect.upper() == "SNOWFLAKE":
+    if str(dialect).upper() == "SNOWFLAKE":
```

Comment:
Refactor this to use `SQLGLOT_DIALECTS` from `config.py`

Reply:
Done